lib/index.ts

```diff
@@ -276,10 +276,14 @@ export default class Backburner {
       }
     }
 
-    if (length === 1) {
-      return method();
-    } else if (length === 2) {
-      return method.call(target);
+    let onError = getOnError(this.options);
+
+    if (onError) {
+      try {
```

Comment:
Previously we had kept the try/catch in its own method, in-order to not deopt the entire function.

v8 turbofan now no longer deopts the function if try/catch is present, but other runtimes and older browser (on slow android etc) may still have this issue.

Thoughts?